connectors/sources/s3.py

```diff
@@ -110,7 +110,7 @@ async def _get_content(self, doc, region, timestamp=None, doit=None):
         if os.path.splitext(filename)[-1] not in TIKA_SUPPORTED_FILETYPES:
             logger.debug(f"{filename} can't be extracted")
             return
-        if doc["size"] > DEFAULT_MAX_FILE_SIZE:
+        if doc["size (bytes)"] > DEFAULT_MAX_FILE_SIZE:
```

Comment:
let's go for `size_in_bytes` -- I don't want to introduce a filed name that's a complex description